femtools/Solvers.F90

```diff
@@ -1953,6 +1953,8 @@ recursive subroutine setup_pc_from_options(pc, pmat, option_path, &
         ! we think this is a more useful default - the default value of 0.0
         ! causes spurious "unsymmetric" failures as well
         call PCGAMGSetThreshold(pc, 0.01, ierr)
+        ! this was the old default:
+        call PCGAMGSetCoarseEqLim(pc, 800, ierr)
```

Comment:
Would be good to  have a better explanation for why this is the "right" number.  Is it worth adding it as an option or doing more tests?